AABB_tree/include/CGAL/internal/AABB_tree/AABB_ray_intersection.h

```diff
@@ -187,10 +187,11 @@ class AABB_ray_intersection {
       typename AABB_traits::Geom_traits::Vector_3 v = ray->to_vector();
 
       for(int i = 0; i < 3; ++i) {
-        if(v[0] != FT(0.)) {
-          return x[0] / v[0];
+        if(v[i] != FT(0.)) {
+          return x[i] / v[i];
         }
```

Comment:
Well, `i` is actually used: as the loop counter, in the expressions `i < 3` and `++i`. That is probably why no compiler warns about `i` being unused.